### DISCUSSION TITLE
OHLC Chart

### DISCUSSION THREAD

**Discussion by ghost** [has2k1/plotnine#884] [Q&A] [answered]
```
`p2 = (
p9.ggplot(df, aes(x = "date")) +
p9.geom_segment(aes(y = "low", yend = "high", xend = "date"), color = "black") +
p9.geom_segment(aes(y = "open", yend = "open", xend = "date - 0.1"), color = "black") +
p9.geom_segment(aes(y = "close", yend = "close", xend = "date + 0.1"), color = "black") +
p9.labs(title = "OHLC Chart", x = "Date", y = "Price") +
p9.theme_minimal()
)

p2.show()`
```

When given a DataTimeArray as "date", and needing to compute an offset from the axis coordinate, i.e. "date - 0.1", the following error occurs:: 'DatetimeArray' and 'float'

Can you please assist with the proper formatting for xend in this case.

Thanks!

Sorry - I wrongfully submitted this question to "Issues"

**Comment by TerryGamon** [comment, marked as answer]
https://github.com/has2k1/plotnine/issues/883#issuecomment-2453442632
